README.md

```diff
@@ -39,8 +39,10 @@ Projects listed on this page are open source that Microsoft maintain or contribu
+|                                      | [Gateway API](https://gateway-api.sigs.k8s.io/)                                                                                                                                                                                                                          | A sig-network subproject that establishes a specification for service networking in Kubernetes (ingress and service mesh)                                                                                                                                                                                                                         | Kubernetes: v1beta <br> APIs: [Standard and Experimental](https://gateway-api.sigs.k8s.io/concepts/versioning/?h=version#release-channels)| [Istio Slack](https://slack.istio.io/) <br> [GitHub issues](https://github.com/istio/istio/issues)                                                                                                                                                                                                                                                                          | [Use application gateway for containers](https://learn.microsoft.com/en-us/azure/application-gateway/for-containers/overview)                                                                                                                                                                                                                                                                                                                              |
```

Comment:
should the slack link for gateway api be https://kubernetes.slack.com/archives/CR0H13KGA as seen here: https://gateway-api.sigs.k8s.io/contributing/#communications

Reply:
Ah I missed that; sending a follow up